I-don't-have-Vinegrate.py

```diff
@@ -0,0 +1,68 @@
+import random
+import datetime
+import calendar
+
+"""
+This program gets from the user two dates and generates a date between the two date. If the generated
+date is Monday- the program will print 'I am out of Vinegrate', otherwise it'll print the day of that
+date.
+Note- you can only enter dates after the first unix epoch second which was at- 00:00:00 UTC on 1 January 1970
+"""
+
+# This function converts the random timestamp to a date and prints the day of the week of the
```

Comment:
This is not the way you document python functions -
![Capture](https://user-images.githubusercontent.com/13467749/160239965-cfec0718-1069-4159-b4dd-e620d22be1d7.JPG)
Also, you can just use the pycharm autocomplete